react-cors-spa/package.json

```diff
@@ -5,19 +5,22 @@
   "email": "jguilloi@amazon.fr",
   "license": "MIT",
   "devDependencies": {
+    "@tanstack/eslint-plugin-query": "^5.53.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "react-scripts": "^5.0.1"
   },
   "dependencies": {
-    "axios": "^1.7.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "@tanstack/react-query": "^5.54.1",
+    "@tanstack/react-query-devtools": "^5.54.1",
```

Comment:
Should be a dev dependency

Reply:
https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencie

Reply:
For react query and for react query devtools?

Or soley for devtools?

Reply:
Just the devtools